scripts/modules/cli.py

```diff
@@ -88,8 +88,8 @@ class LocalSetup(object):
         '8.5': '8.5.3',
         '8.6': '8.6.1',
         # UPDATECLI_AUTOMATION
-        'main': '8.7.0',
-        'master': '8.7.0',  # keep master alias for backward compatibility. Upgrade the main alias only
+        "main": "8.7.0",
```

Comment:
I was forced to use double quotes, for some reason the `updatecli` does not work well with single quotes in the `replacepattern`